data/libs/SpaceStation.lua

```diff
@@ -191,53 +223,73 @@ local equipmentPrice = {}
 --
 -- Method: GetEquipmentPrice
 --
--- Get the price of an equipment or cargo item traded at this station
+-- Get the price of an equipment item traded at this station
 --
 -- > price = station:GetEquipmentPrice(equip)
 --
 -- Parameters:
 --
---   equip - the <Constants.EquipType> string for the equipment or cargo item
+--   equip - the <Constants.EquipType> string for the equipment item
 --
 -- Returns:
 --
---   price - the price of the equipment or cargo item
+--   price - the price of the equipment item
 --
 -- Availability:
 --
 --   alpha 10
 --
 -- Status:
 --
---   experimental
+--   stable
 --
-
 function SpaceStation:GetEquipmentPrice (e)
 	assert(self:exists())
-	if not equipmentPrice[self] then equipmentPrice[self] = {} end
-	if equipmentPrice[self][e] then
-		return equipmentPrice[self][e]
+
+	if equipmentPrice[self] then
+		return equipmentPrice[self][e] or e.price
 	end
-	local mul = e:IsValidSlot("cargo") and ((100 + Game.system:GetCommodityBasePriceAlterations(e.name)) / 100) or 1
-	return mul * e.price
+
+	return e.price
 end
 
-function SpaceStation:SetEquipmentPrice (e, v)
+--
+-- Method: SetEquipmentPrice
+--
+-- Set the price of an equipment item traded at this station
+--
+-- > station:SetEquipmentPrice(equip, price)
+--
+-- Parameters:
+--
+--   equip - the <Constants.EquipType> string for the equipment item
+--
+--   price - the new price of the equipment item
+--
+-- Availability:
+--
+--   alpha 10
+--
+-- Status:
+--
+--   stable
+--
+function SpaceStation:SetEquipmentPrice (e, price)
```

Comment:
it doesn't seem to be used anywhere? (as well as the table `equipmentPrice`)

Reply:
It is indeed not currently used anywhere but is part of existing public API so I intend to keep it around for future modules to use (e.g. dynamic equipment discounts on certain stations, "easter egg" station/system which sells all equipment for 25% off, etc).